beacon-chain/blockchain/execution_engine.go

```diff
@@ -274,7 +274,7 @@ func (s *Service) pruneInvalidBlock(ctx context.Context, root, parentRoot, lvh [
 	}
 }
 
-// getPayloadAttributes returns the payload attributes for the given state and slot.
+// getPayloadAttribute returns the payload attributes for the given state and slot.
```

Comment:
The function above on line 256 the comment is not related to the function.
`// reportInvalidBlock deals with the event that an invalid block was detected by the execution layer` should be modified to reflect the function itself